### DISCUSSION TITLE
Enable sponsors

### DISCUSSION THREAD

**Discussion by Kreijstal** [lshqqytiger/ZLUDA#12] [Ideas]
is it possible to enable sponsorship?

**Comment by lshqqytiger** [comment]
This is just a fork of ZLUDA. I don't receive any aponsorship/donation.
related discussion on the upstream: https://github.com/vosen/ZLUDA/issues/57

**Reply by Kreijstal** [reply]
Is there anyone what wants sponsors then?

**Reply by erkinalp** [reply]
@lshqqytiger upstream stopped development due to legal threats

**Reply by Kreijstal** [reply]
from whom, AMD? lmao

**Reply by erkinalp** [reply]
yeah, he reverted to pre-AMD code and now developing the AMD code again

**Comment by radna0** [comment]
If I could I would donate my money as well, I'm really trying to make Zluda and Pytorch to work so that I could use it for deep learning workloads on AMD GPUS

**Reply by Kreijstal** [reply]
someone that wants to sponser themselves could very well do it

**Reply by radna0** [reply]
I understand what you're saying. My focus right now is on getting Zluda and PyTorch to work together for deep learning on AMD GPUs, which is where my resources are currently going. Once I achieve that, hopefully publishing the result would be beneficial for the whole community. I really don't want to choose the Nvidia route because there is so much potential with AMD GPUs
